Components/NetworkAddons/Connections/Resend/CCResendConnection.h

```diff
@@ -12,11 +12,21 @@
 #import <Foundation/Foundation.h>
 #import "TRCConnectionProxy.h"
 
+typedef struct {
+    BOOL shouldResend;
+    NSTimeInterval delay;
+    NSInteger maxResendAttempts;
+} CCResendDecision;
+
 extern NSString *CCResendConnectionShouldResend;
```

Comment:
Should be `const`, like
`extern NSString * const CCResendConnectionShouldResend;`